docs/_sass/mailing-list.scss

```diff
@@ -0,0 +1,61 @@
+.mailing-section {
+    background: #1e2117;
+    color: white;
+    
+
+    a, a:hover, a:focus, a:active{
+        text-decoration: none;
+        color: inherit;
+    }
+    
+    
+    .mailinglist-main {
+        margin: 62px auto;
+        position: relative;
+        display: flex;
+   flex-direction: column;
+    align-items: center;
+
+        .mailinglist-text {
+        
+            
+            margin-bottom: 40px;
+            display: block;
+            span{
+                font-size: 34px;
+            }
+
+            p {
+                max-width: 375px;
+                margin: 0 3rem;
+                color: white;
+                font-size: 20px;
+                
+                
+            }
+        }
+
+        .mailinglist-buttons {
+            display: inline-flex;
+            flex-wrap: wrap;
+            justify-content: center;
+            align-items: center;
+            
+            .mailinglist-join-button {
+                display: flex;
+                justify-content: center;
+                align-items: center;
+                background: #647881;
+                width: 253px;
+                height: 56px;
+                border-radius: 5px;
+                margin: 2rem 0 0;
+                
+            }
+
+            .mailinglist-join-button:not(:first-child) {
+                margin-left: 1rem;
```

Comment:
Remove this left margin for the mobile view using media queries

Reply:
I didn't receive designs for mobile view yet so it's not responsive